src/providers/FileSystemProvider/FileSystemProvider.ts

```diff
@@ -405,7 +400,9 @@ export class FileSystemProvider implements vscode.FileSystemProvider {
   }
 
   public createDirectory(uri: vscode.Uri): void | Thenable<void> {
-    uri = redirectDotvscodeRoot(uri);
+    if (uri.path.includes(".vscode/")) {
```

Comment:
This can probably be removed since we don't actually call to the server to create a directory.

Reply:
In that case I suggest it gets removed to avoid unnecessary changes.